### ISSUE TITLE
Bug: Pipette collides in trash corner during calibration

### ISSUE THREAD

**Issue by Avalei** [Opentrons/opentrons#2663]
## Tech Environment

Robot type: OT-2
App version: 3.5.1 (3.5.1.12363)
Robot software version (if OT2): edge-6168d32

Operating system and version: MacOS Mojave 10.14.1

Protocols used: "[Opentrons Logo Protocol](https://protocols.opentrons.com/protocol/Opentrons_Logo)"
```
"""
@author Opentrons
@date April 27th, 2018
@version 1.3
"""
from opentrons import labware, instruments


def run_custom_protocol(pipette_type: 'StringSelection...'='p300-Single',
    dye_labware_type: 'StringSelection...'='trough-12row'):
    if pipette_type == 'p300-Single':
        tiprack = labware.load('tiprack-200ul', '1')
        pipette = instruments.P300_Single(
            mount='right',
            tip_racks=[tiprack])
    elif pipette_type == 'p50-Single':
        tiprack = labware.load('tiprack-200ul', '1')
        pipette = instruments.P50_Single(
            mount='right',
            tip_racks=[tiprack])
    elif pipette_type == 'p10-Single':
        tiprack = labware.load('tiprack-10ul', '1')
        pipette = instruments.P10_Single(
            mount='right',
            tip_racks=[tiprack])

    if dye_labware_type == 'trough-12row':
        dye_container = labware.load('trough-12row', '2')
    else:
        dye_container = labware.load('tube-rack-2ml', '2')

    output = labware.load('96-flat', '3')
    # Well Location set-up
    dye1_wells = ['A5', 'A6', 'A8', 'A9', 'B4', 'B10', 'C3', 'C11', 'D3',
                  'D11', 'E3', 'E11', 'F3', 'F11', 'G4', 'G10',
                  'H5', 'H6', 'H7', 'H8', 'H9']

    dye2_wells = ['C7', 'D6', 'D7', 'D8', 'E5', 'E6', 'E7', 'E8',
                  'E9', 'F5', 'F6', 'F7', 'F8', 'F9', 'G6', 'G7', 'G8']

    dye2 = dye_container.wells('A1')
    dye1 = dye_container.wells('A2')

    pipette.distribute(
        50,
        dye1,
        output.wells(dye1_wells),
        new_tip='once')
    pipette.distribute(
        50,
        dye2,
        output.wells(dye2_wells),
        new_tip='once')


run_custom_protocol(**{'pipette_type': 'p300-Single', 'dye_labware_type': 'trough-12row'})
```

and the demo script located under the "How it looks" section on [docs.opentrons.com](https://docs.opentrons.com/) API page:
```from opentrons import labware, instruments
plate = labware.load('96-flat', '2')
tiprack = labware.load('tiprack-200ul', '1')
pipette = instruments.P300_Single(mount='left', tip_racks=[tiprack])
pipette.transfer(100, plate.wells('A1'), plate.wells('B2'))
```

## Current Behavior

Running the Opentrons Logo Protocol downloaded off the OpenTrons website, we executed this protocol several times using the P300 single channel pipette as stated as the first pipette option in the protocol. The first time we ran the protocol the pipette gently touched the 4 sides of the black corner in the trash area (with the trash container removed), touched the middle, then stopped in the home position to wait for labware prompts in the app. - we are expecting this to be normal behavior.

The second and proceeding times we attempted to run this protocol the pipette tip would descend far too low in the trash container area and when it went to touch the black corner it ended up snapping the pipette tip off. Again, this is occurring just during the initial calibration steps before actually running the protocol.

We hypothesized that there may have been a mount location issue (being either `right` or `left`) but after testing the two protocols mentioned that each require a different mount location we don't think this is the source of the problem. **Note**: For each of the protocols the colision was **not** seen during the first calibration, however it was seen on subsequent attempts afterwards.

## Expected Behavior
As mentioned above, we are under the impression that the expected behavior is what we saw during the first calibration - after attaching a pipette tip as prompted, the pipette will move to the trash area, descended and lightly touch the 4 sides of the black area, then return to the home position. At that point the app will proceed to prompt us to insert labware on the deck.

## Steps to Reproduce

Please see attached video. [Pipette_colision_in_trash_during_calibration.mov.zip](https://github.com/Opentrons/opentrons/files/2573870/Pipette_colision_in_trash_during_calibration.mov.zip)

This behavior was repeated several times, we only tested this with the P300 and then stopped for fear of damaging the pipette.

1. Start the "[Opentrons Logo Protocol](https://protocols.opentrons.com/protocol/Opentrons_Logo)" with the OpenTrons app with a P300 single channel pipette installed
2. Click the "Proceed to Calibrate" button
3. When prompted by the app attach a compatible pipette tip by hand
4. Click "Continue" 
5. Observe the X and Z distances that cause the pipette tip to snap off during the pipette movment
6. If the tip collision does not occur on the first calibration attempt, try to run the protocol again after it as completed the first time

**Comment by btmorr** [comment]
Hi @Avalei, thanks for the bug report. Would you mind checking a couple of options for me on your robot?

First, in the app when you connect to the robot, scroll down and you should see the "Advanced Settings" section. Can you check if the "Short (55mm) fixed-trash" setting is enabled"? Your robot appears to have the tall trash, so if that toggle is enabled it could cause the behavior you are seeing.

For recovery, in the app you can click on the "Reset" button and select "Tip Length". This should put your robot back into a state prior to probing, which seems to be saving incorrect data on your robot. Please re-try the tip length probe again after resetting, and see if the behavior persists.

If the short trash option is not enabled and resetting the tip length does not fix it, I'll continue trying to discover what is failing. Thank you!

**Comment by Avalei** [comment]
Hi @btmorr, thanks for your quick help! 

It looks like pressing the "Reset" button did the trick. More specifically we had to actually turn the machine off, then on, then press reset. We had tried a couple times simply hitting reset and it didn't fix the problem.

I'll close this issue out now since the calibration and protocols seem to be running properly now with no more collision seen. Thanks again!
